Build failure on Windows

Non-unity builds seems to fail for me on Windows, MSVC 2019 with the following error:
```
3>------ Build started: Project: common, Configuration: Debug Win32 ------
3>Automatic MOC for target common
3>TextureCollectionLoader.cpp
3>C:\dev\TrenchBroom\common\src\IO\TextureCollectionLoader.h(43,119): error C2039: 'string': is not a member of 'std'
```

I'm not sure if this affects unity builds as I'm not using it due to older CMake.

--- a/common/src/IO/TextureCollectionLoader.h
+++ b/common/src/IO/TextureCollectionLoader.h
@@ -25,6 +25,7 @@
 
 #include <memory>
 #include <vector>
+#include <string>
 
 namespace TrenchBroom {
     class Logger;

